hilti/toolchain/src/compiler/driver.cc

```diff
@@ -480,12 +480,14 @@ Result<Nothing> Driver::addInput(const std::filesystem::path& path) {
             _addUnit(std::move(*unit));
         }
 
+        _need_jit = true;
```

Comment:
I feel this mutation would really fit better in `_addUnit`, but on the other hand we don't have `_external_cxxs` not as well encapsulated, so that would introduce an inconsistency.

Reply:
Good point, moving it there (while leaving it with _external_cxxs)